apollo-router/src/plugins/expose_query_plan.rs

```diff
@@ -218,6 +218,10 @@ mod tests {
             build_mock_supergraph(serde_json::json! {{
                 "plugins": {
                     "experimental.expose_query_plan": true
+                },
+                "supergraph": {
+                    // TODO(@goto-bus-stop): need to update the mocks and remove this, #6013
```

Comment:
should this be linking to a new issue (6013 is the existing PR#)?

Reply:
The work would likely be done as part of removing `reuse_query_fragments`, I linked the PR as a place where you can find context